src/screens/CatalogScreen.js

```diff
@@ -1,28 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
   StyleSheet, SafeAreaView, StatusBar, ScrollView,
 } from 'react-native';
+import { connect } from 'react-redux';
 
+import { shape } from 'prop-types';
 import TimelineWidget from '../components/Timeline/TimelineWidget';
 import CategorySelector from '../components/CategorySelector/CategorySelector';
-import RecordCardsContainer from '../components/RecordCard/RecordCardsContainer';
+import RecordCardsContainer from '../components/SubTypeAccordion/SubTypeAccordionsContainer';
 import Colors from '../constants/Colors';
 import FilterDrawer from '../components/FilterDrawer/FilterDrawer';
 
-const CatalogScreen = () => (
-  <SafeAreaView style={styles.safeAreaView}>
-    <StatusBar backgroundColor={Colors.primary} barStyle="dark-content" />
-    <FilterDrawer>
-      <ScrollView style={styles.content}>
-        <TimelineWidget />
-        <CategorySelector />
-        <RecordCardsContainer />
-      </ScrollView>
-    </FilterDrawer>
-  </SafeAreaView>
-);
+const CatalogScreen = ({ resourceIdsGroupedByType }) => {
+  const categories = Object.keys(resourceIdsGroupedByType);
+  const scrubbedCategories = categories.filter((category) => category !== 'Patient' && category !== 'Observation');
+  const [selectedCategory, setSelectedCategory] = useState(scrubbedCategories[0]);
 
-export default CatalogScreen;
+  return (
+    <SafeAreaView style={styles.safeAreaView}>
+      <StatusBar backgroundColor={Colors.primary} barStyle="dark-content" />
+      <FilterDrawer>
+        <ScrollView>
+          <TimelineWidget />
+          <CategorySelector
+            categories={scrubbedCategories}
+            selectedCategory={selectedCategory}
+            setSelectedCategory={setSelectedCategory}
+          />
+          { selectedCategory && (
+            <RecordCardsContainer
+              selectedCategory={selectedCategory}
+            />
+          )}
+        </ScrollView>
+      </FilterDrawer>
+    </SafeAreaView>
+  );
+};
+
+CatalogScreen.propTypes = {
+  resourceIdsGroupedByType: shape({}).isRequired,
+};
+
+const mapStateToProps = (state) => ({
+  resourceIdsGroupedByType: state.resourceIdsGroupedByType,
```

Comment:
See: `src/screens/SummaryScreen.js`: 
This should be using the existing `supportedResourcesSelector(state)`  (which already filters supported categories, _and sorts_ them by their label)

...if it lacks the necessary structure substructure, a new selector can be created, derived from `supportedResourcesSelector`

Reply:
this is addressed in #29